ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/minimal-permission/cucushift-installer-rehearse-gcp-ipi-minimal-permission-workflow.yaml

```diff
@@ -8,6 +8,7 @@ workflow:
     env:
       CONTROL_PLANE_NODE_TYPE: "custom-4-16384"
       COMPUTE_NODE_TYPE: "custom-4-16384"
+      GCP_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
```

Comment:
If we already inject `gcp-provision-minimal-permission` into most of install workflow, I do not think we need this dedicated minimal-permission install workflow any more.

Reply:
+1 , and even the `cucushift-installer-rehearse-gcp-ipi-minimal-permission-provision` & `cucushift-installer-rehearse-gcp-ipi-minimal-permission-deprovision` chains, as we already implement the necessary steps in vaiours provision chain and cucushift-installer-rehearse-gcp-ipi-deprovision chain.

Reply:
FYI
1. I'd updated chain `cucushift-installer-rehearse-gcp-ipi-provision`, to include the additional steps of chain `cucushift-installer-rehearse-gcp-ipi-minimal-permission-provision`. 
2. I'd removed the reference of workflow `cucushift-installer-rehearse-gcp-ipi-minimal-permission` from 4.19 tests, which instead use workflow `cucushift-installer-rehearse-gcp-ipi`. 
3. I will delete the workflow `cucushift-installer-rehearse-gcp-ipi-minimal-permission`, after removing its references in other versions tests.